outscale/resource_outscale_volume.go

```diff
@@ -306,9 +391,14 @@ func readOAPIVolume(d *schema.ResourceData, volume oscgo.Volume) error {
 			return err
 		}
 	}
-
-	if err := d.Set("iops", volume.GetIops()); err != nil {
-		return err
+	if volume.GetVolumeType() != "standard" {
+		if err := d.Set("iops", volume.GetIops()); err != nil {
+			return err
+		}
+	} else {
+		if err := d.Set("iops", 150); err != nil {
```

Comment:
use constant instead of hard coded value. Maybe a link to the doc also for future update